draft-ietf-core-yang-cbor.md

```diff
@@ -1597,7 +1597,7 @@ CBOR diagnostic notation (the line break is inserted for exposition only):
 
 ~~~~ CBORdiag
 "/ietf-system:system/authentication/user[name='bob']/
-authorized-key[name='admin']/key-data"
+authorized-key[name='admin', country='CA']/key-data"
 ~~~~
 
 CBOR encoding:
```

Comment:
Please also fix the `CBORbytes`. I believe it should be

```
78 67 2F696574662D73797374656D3A73797374656D2F61757468656E7469636174696F6E2F757365725B6E616D653D27626F62275D2F617574686F72697A65642D6B65795B6E616D653D2761646D696E272C20636F756E7472793D274341275D2F6B65792D64617461
```

Reply:
As I said in #43, the keys of authorized-key are name and not country, so I don't know why this change is needed and what it would mean.